### PR TITLE
Create vicenza.md

### LINKED ISSUE #21: Another hometown profile
So here's a hometown profile.....

### PR DESCRIPTION
**Pull request by ilaria-fanton**
This pull request closes #21

### CODE REVIEW

File: vicenza.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## Visiting Vicenza - Italy
```
Comment (hfgnr) [inline vicenza.md:1]: Where is this in Italy?

File: vicenza.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+### Fun things to visit
```
Comment (hfgnr) [inline vicenza.md:3]: Sad that there isn't anything :(

**Comment by hfgnr** [comment]
Please let me know more detail on this - I love Italy!

File: vicenza.md (hunk 1 above)
Comment (tobiasbinz) [inline vicenza.md:1]: Italy is in the South of Europe

**Pull request merged**